sklego/loess.py

```diff
@@ -0,0 +1,332 @@
+import logging
+from math import floor
+
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.base import BaseEstimator
+from sklearn.base import TransformerMixin
+from sklearn.linear_model import LinearRegression
+from sklearn.neighbors import NearestNeighbors
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import PolynomialFeatures
+
+
+class LoessSmoother(TransformerMixin, BaseEstimator):
+    def __init__(self,
+                 model=None,
+                 n_degree=2,
+                 transformer_type=True,
+                 window_method='fixed',
+                 window_size=2,
+                 step_size=.2,
+                 fraction=.1):
+        """
+
+        Loess Regression. This class implements multiple methods of Loess regression for the single
```

Comment:
![image](https://user-images.githubusercontent.com/1019791/55324912-bd9ad280-5483-11e9-9099-c2f3389d8a67.png)

not that i want to make this a *super* important thing, but could you make the width a bit smaller of the docstring here? makes reviewing in smaller screens easier.